aries/aries_vcx_core/src/anoncreds/credx_anoncreds/mod.rs

```diff
@@ -130,7 +130,7 @@ impl RecordWallet for WalletAdapter {
     async fn search_record(
         &self,
         category: RecordCategory,
-        search_filter: Option<SearchFilter>,
+        search_filter: Option<String>,
```

Comment:
At glance seem to be unrelated with the nature of the PR. Why was this changed?

Reply:
Ah I see, the 2 wallet implementations use different types for search and there's some feature flagging to put them both under `SearchFilter` type. 
Yeah we can address that in the future. Resolving